packages/adapter-static/README.md

```diff
@@ -61,6 +61,34 @@ export default {
 
 When operating in SPA mode, only pages that have the [`prerender`](https://kit.svelte.dev/docs#ssr-and-javascript-prerender) option set will be prerendered.
 
+## Github Pages
+
+When building for Github Pages, make sure to update the [basepaths](https://kit.svelte.dev/docs#configuration) (see `paths.base` and `paths.assets`).
+
+Also note that you will have to prevent Github's provided Jekyll from managing your site by putting an empty `.nojekyll` file in your static folder. If you do not want to disable Jekyll, change the kit's `appdir` configuration option to `"app_"` or anything not starting by an underscore. For more information, see Github's documentation about [configuring jekyll](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll#configuring-jekyll-in-your-github-pages-site).
+
+A config for Github Pages might look like:
+
+```js
+/** @type {import('@sveltejs/kit').Config} */
+const config = {
+	...
+	kit: {
+		...
+		// Since your Github Pages' default url will be https://your-username.github.io/your-repo-name,
+		// you'll have the change the basepaths.
+		// Note that you may have to comment these in dev mode.
```

Comment:
This seems like quite the caveat. I don't know if we'd want to promote this solution without making it a bit nicer to use first

Reply:
paths are currently broken in dev mode. i'll try to review https://github.com/sveltejs/kit/pull/2145 and then we can see if that changes the situation

Reply:
other option is to detect the dev environment: 
```js
const isProduction = process.env.NODE_ENV === 'production';

...
    paths: {
        base: isProduction ? "project_name" : "",
    }
...
```